configs/projects/puppet-agent.rb

```diff
@@ -106,17 +106,17 @@
   # Platform specific
   if platform.is_windows?
     arch = platform.architecture == "x64" ? "64" : "32"
-    proj.setting(:cflags, "-IC:/tools/pl-build-tools/include -IC:/tools/mingw#{arch}/include -I#{platform.convert_to_windows_path(proj.includedir)}")
-    proj.setting(:ldflags, "-LC:/tools/pl-build-tools/lib -LC:/tools/mingw#{arch}/lib -L#{platform.convert_to_windows_path(proj.libdir)}")
-    proj.setting(:gcc_bindir, "#{platform.drive_root}/tools/mingw#{arch}/bin")
+    proj.setting(:gcc_root, "C:/tools/mingw#{arch}")
+    proj.setting(:gcc_bindir, "#{proj.gcc_root}/bin")
+    proj.setting(:cc, "#{proj.gcc_bindir}/gcc")
+    proj.setting(:cxx, "#{proj.gcc_bindir}/g++")
```

Comment:
We probably don't need to set these at all, but I like having them around just in case... What do other people think? We're not using `:cc` or `:cxx` anywhere currently. Should I take them out? We can always add them back in if we decide we don't need them